docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -174,23 +172,22 @@ dune --create-account testaccount
 dune --send-action hello test '[bob]' testaccount
 ```
 
-You should see a transaction executed successfully on the first time, and if you try to repeat this command it will 
-fail because that row already exists in the contract's database.
+You should see a transaction executed successfully with a row added to the contract's database. If you repeat this command it will fail because that row already exists in the contract's database.
 
 ### Get data from your contract
 
-You just added a row to the contract's database, let's fetch that data from the chain:
+You just added a row to the contract's database. You can fetch that data from the chain:
 
 ```shell
 # format: dune --get-table <CONTRACT> <SCOPE> <TABLE>
 dune --get-table hello hello users
 ```
 
-You should get a table result with one or more row. If you didn't make sure your interaction above was successful.
+A table results with one or more rows. If you did not receive a table with one or more rows, make sure the interaction above was successful.
```

Comment:
"A table results" doesn't sound right, maybe "You get a table result..."?

Reply:
Done